index.js

```diff
@@ -25,7 +26,8 @@ function fastifyCors (fastify, opts, next) {
     allowedHeaders: null,
     maxAge: null,
     preflight: true,
-    hideOptionsRoute: true
+    hideOptionsRoute: true,
+    strictPreflight: false
```

Comment:
I wonder if we could add a deprecation notice to indicate that this default will change to `true` in the next version.